src/pages/workspace/companyCards/utils.tsx

```diff
@@ -48,49 +47,82 @@ function getExportMenuItem(
+                            companyCard?.nameValuePairs?.quickbooks_online_export_account === CONST.COMPANY_CARDS.DEFAULT_EXPORT_TYPE)
+                    );
+                    title = isDefaultTitle ? defaultCard : companyCard?.nameValuePairs?.quickbooks_online_export_account;
+                    selectedAccount = companyCard?.nameValuePairs?.quickbooks_online_export_account ?? defaultAccount;
+                    exportType = CONST.COMPANY_CARDS.EXPORT_CARD_TYPES.NVP_QUICKBOOKS_ONLINE_EXPORT_ACCOUNT;
                     break;
-                case CONST.QUICKBOOKS_NON_REIMBURSABLE_EXPORT_ACCOUNT_TYPE.DEBIT_CARD:
+                }
+                case CONST.QUICKBOOKS_NON_REIMBURSABLE_EXPORT_ACCOUNT_TYPE.DEBIT_CARD: {
                     data = quickbooksOnlineBankAccounts ?? [];
-                    title = companyCard?.nameValuePairs?.quickbooks_online_export_account_debit ?? nonReimbursableExpensesAccount?.name;
+                    isDefaultTitle = !!(
+                        defaultAccount &&
+                        (!companyCard?.nameValuePairs?.quickbooks_online_export_account_debit ||
+                            companyCard?.nameValuePairs?.quickbooks_online_export_account_debit === CONST.COMPANY_CARDS.DEFAULT_EXPORT_TYPE)
+                    );
+                    title = isDefaultTitle ? defaultCard : companyCard?.nameValuePairs?.quickbooks_online_export_account_debit;
+                    selectedAccount = companyCard?.nameValuePairs?.quickbooks_online_export_account_debit ?? defaultAccount;
                     exportType = CONST.COMPANY_CARDS.EXPORT_CARD_TYPES.NVP_QUICKBOOKS_ONLINE_EXPORT_ACCOUNT_DEBIT;
                     break;
+                }
                 default:
                     shouldShowMenuItem = false;
                     data = [];
             }
 
+            const resultData = data.length > 0 ? [...data, defaultMenuItem] : data;
+
             return {
                 description,
                 title,
                 exportType,
                 shouldShowMenuItem,
                 onExportPagePress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_EXPORT.getRoute(policyID)),
-                data: data.map((card) => ({
+                data: resultData.map((card) => ({
                     value: card.name,
                     text: card.name,
                     keyForList: card.name,
-                    isSelected: card.name === title,
+                    isSelected: isDefaultTitle ? card.name === defaultCard : card.name === selectedAccount,
                 })),
             };
         }
         case CONST.POLICY.CONNECTIONS.NAME.XERO: {
             const type = translate('workspace.xero.xeroBankAccount');
             const description = currentConnectionName && type ? translate('workspace.moreFeatures.companyCards.integrationExport', {integration: currentConnectionName, type}) : undefined;
```

Comment:
In the mock data, currentConnectionName is `Xero`, type is `Xero bank account` --> description will have 2 Xero word

<img width="846" alt="Screenshot 2024-11-22 at 15 08 46" src="https://github.com/user-attachments/assets/a4b7b91f-9e69-46f5-8dc6-64834abaa78e">